catalog/model/src/main/java/org/projectnessie/catalog/model/ops/CatalogOperationTypeResolver.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.projectnessie.catalog.model.ops;
 
-import javax.annotation.Nonnull;
+import jakarta.validation.constraints.NotNull;
```

Comment:
Why switch to jakarta here? Is `javax.annotation.Nonnull` not standard enough?

Reply:
It'll be Java 11 soon-ish anyways. We don't have a "public catalog API" anyways.